app/controllers/recipes_controller.rb

```diff
@@ -3,16 +3,21 @@ class RecipesController < ApplicationController
 
   # GET /recipes or /recipes.json
   def index
+    # Should only show recipes from the logged in user
+    # Leads to the recipes details
     @recipes = Recipe.all
   end
 
   # GET /recipes/1 or /recipes/1.json
   def show
+    # Should allow the user to delete it's own recipes
+    # Show details of the recipe
     @recipe = Recipe.find(param[:id])
   end
 
   # GET /recipes/new
   def new
+    # The logged-in user should be able to create a new recipe
     @recipe = Recipe.new
   end
 
```

Comment:
I am getting this error when I try to create a new recipe.
I think the recipes controller create function should create based on users.
```
user = current_user
    @recipe = Recipe.new(recipe_params)
    @recipe.user = user
```
![Screenshot 2022-07-28 100408](https://user-images.githubusercontent.com/77745775/181442607-6721d0fc-9578-4a38-910d-f20d604ae221.png)

Reply:
No, you can't rught now. I nade that pull request just to be able to use the same routes as you and the design.